src/platform_impl/web/window.rs

```diff
@@ -153,6 +153,16 @@ impl Window {
         // Intentionally a no-op: users can't resize canvas elements
     }
 
+    #[inline]
+    pub fn resize_increments(&self) -> Option<PhysicalSize<u32>> {
+        None
+    }
+
+    #[inline]
+    pub fn set_resize_increments(&self, _increments: Option<Size>) {
+        // Intentionally a no-op: users can't resize canvas elements
```

Comment:
ditto about warning in log.

Reply:
In those new dummy methods for other platforms I replicated surrounding comments/warnings/whatever - so that they look consistent with everything else that's already there